.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.1.0/schema.json",
+  "changelog": "@changesets/changelog-git",
+  "commit": true,
```

Comment:
Might want to remove this in light of issues with auto generated commits: https://github.com/guardian/source/pull/1521